### ISSUE TITLE
add an option :use `space key` to  jump into tag

### ISSUE THREAD

**Issue by u-Sir** [adirh3/Fluent-Search#445]
**Is your feature request related to a problem? Please describe.**
when putting our hands on the keyboard, thumb is above the space key. `space` is more easy to reach than `tab`. And then FS can use `tab` to select the result.

**Describe the solution you'd like**
add an option :use `space key` to  jump into tag, like this:
![图片](https://user-images.githubusercontent.com/93647031/149650139-12e20ef1-896c-461a-933b-c19b0900f9d8.png)

**Describe alternatives you've considered**

**Additional context**

**Comment by Metroid-Prime** [comment]
We had this before -- space was the default trigger key for stuff like web searches.
Adir removed it because it was confusing users . Although I totally understand that, this is my most desired request.  I just can't get used to the tab feature and I've tried.  I've completed about 300 web searches in a week and I find the tab key to be more cumbersome to use.    

Normally I try to shy away from polluting settings with extraneous entries, but this is important enough to the user experience that I wish there was.  Maybe select which key you want,   tab, space or other keys  and set the default to tab for the general user.

@MakeshVineeth @Nomad-0  not sure how you guys feel about this.
